modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc

```diff
@@ -9,9 +9,12 @@ The Subscriber source in the Anypoint MQ connector enables the app to listen for
 
 By default, the Subscriber source works in continuous listening mode, providing maximum throughput by consuming messages as soon as they arrive in the queue. This is called the `prefetch` subscriber-type configuration. When using the `prefetch` mode, the Subscriber source attempts to keep a local buffer of messages full, making them available to be dispatched to the Mule flow as soon as the app can accept them.
 
-For `subscriber-type`, you configure the `maxLocalMessages` parameter to specify the target size for a full buffer, which controls how quickly messages are taken from the queue.
+.The arrow shows the value of the *Subscriber type* field set to *Prefetch* in the *Subscriber* properties window.
+image::anypoint-mq/3.x/amq-3x-subscriber-prefetch.png[Prefetch subscriber type]
+
+
+For the `prefetch` subscriber-type, you configure the `maxLocalMessages` parameter to specify the target size for a full buffer, which controls how quickly messages are taken from the queue.
```

Comment:
Why is "subscriber-type" hyphenated here?
I see that it is earlier, in "`prefetch` subscriber-type configuration," which makes sense as a compound adjective, but I also see that the alt text uses "subscriber type"?